_posts/2020-10-15-2.0.0-release.md

```diff
@@ -0,0 +1,139 @@
+
+## Go notes
+
+## Java notes
+The Java package has supported a number of new features. 
+Users can validate vectors in a wider range of aspects, if they are willing to take more time. 
+In dictionary encoding, dictionary indices can be expressed as unsinged integers.
+A framework for data compression has been setup for IPC.  
+
+The calculation for vector capacity has been simplified, so users should experience notable performance
+improvements for various 'setSafe' methods. 
+
+Bugs for JDBC adapters, sort algorithms, and ComplexCopier have been resolved to make them more usable.
+## JavaScript notes
+
+## Python notes
+
+Parquet reading now supports round trip of arbitrarily nested data. Several bug fixes
+for writing nested data and FixedSizeList.  If writing data with these type we recommend validating 
+old data (there is potential some data loss) and upgrade to 2.0.
+
+Extension types now round trip through Parquet.
+
+Conversion of Timezone aware datetimes to and/from pyarrow arrays including pandas now round-trip preserving timezone.  To use the old behavior (e.g. for spark) set the environment variable PYARROW_IGNORE_TIMEZONE to a truthy value (i.e. 
+`PYARROW_IGNORE_TIMEZONE=1`)
+
+The `pyarrow.filesystem` submodule is deprecated in favor of new filesystem
+implementations in `pyarrow.fs`.
+
+The custom serialization functionality (`pyarrow.serialize()`,
+`pyarrow.deserialize()`, etc) is deprecated. Those functions provided a
+Python-specific (not cross-language) serialization format which were not
+compatible with the standardized Arrow (IPC) serialization format.  
+For arbitrary objects, you can use the standard library ``pickle``
+functionality instead. For pyarrow objects, you can use the IPC
+serialization format through the ``pyarrow.ipc`` module, as explained
+above.
+
+The `pyarrow.compute` module now has a complete coverage of the available C++
+compute kernels in the python API. Several new kernels have been added (... TODO
+list a few ...).
+
+The `pyarrow.dataset` module was further improved. In addition to reading,
+it is now also possible to write partitioned datasets (with `write_dataset()`).
+
+The Arrow <-> Python conversion code was refactored, fixing several bugs
+and corner cases.
+
+Conversion of an array of `pyarrow.MapType` to Pandas has been added.
+
+## R notes
+
+
+For more on what’s in the 2.0.0 R package, see the [R changelog][4].
+
+## Ruby and C GLib notes
+
+### Ruby
+
+In Ruby binding, `Arrow::Table#save` uses the number of rows as the `chunk_size` parameter by default when the table is saved in a Parquet file.
+
+### C GLib
+
+The GLib binding newly supports `GArrowStringDictionaryArrayBuilder` and `GArrowBinaryDictionaryArrayBuilder`.
+
+Moreover the GLib binding supports new accessors of `GArrowListArray` and `GArrowLargeListArray`.  They are `get_values`, `get_value_offset`, `get_value_length`, and `get_value_offsets`.
+
+## Rust notes
+
```

Comment:
@andygrove what's the status of your Rust blog post? If it's not ready right yet, maybe make a note that there's a lot of changes and there will be a followup post specifically about Rust

Reply:
@nealrichardson I just created the PR: https://github.com/apache/arrow-site/pull/81